Content.Shared/Rootable/RootableComponent.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Content.Shared.Alert;
+using Content.Shared.FixedPoint;
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+
+namespace Content.Shared.Rootable;
+
+/// <summary>
+/// A rooting action, for Diona.
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState, AutoGenerateComponentPause]
+public sealed partial class RootableComponent : Component
+{
+    /// <summary>
+    /// The action prototype that toggles the rootable state.
+    /// </summary>
+    [DataField]
+    public EntProtoId Action = "ActionToggleRootable";
+
+    /// <summary>
+    /// Entity to hold the action prototype.
+    /// </summary>
+    [DataField]
```

Comment:
```suggestion
    [DataField, AutoNetworkedField]
```
I think this one has to be networked as well